starter-code/apple-pie-starter-code/index.html

```diff
@@ -2,34 +2,51 @@
 <html lang="en">
 <head>
   <title>Apple Pie Recipe</title>
+  <link rel="stylesheet" type="text/css" href="css/main.css">
 </head>
 <body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
+<div> </div> 
+<h1>Apple Pies</h1>
+
+
+<p>This was my grandmother's apple pie recipe. I have never seen another one
 quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
-
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
-
-Directions
-Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
+prizes in local competitions. I hope it becomes one of your favorites as well!</p>
+
+<hr>
+
+<p> <h3>Ingredients</h3>
```

Comment:
you are not using <div>. This <p> should be a <div> tag. 

<div> are the boxes you put all the content in. It's all about boxes inside boxes inside boxes... :wink: